include/nbl/ext/FFT/FFT.h

```diff
@@ -0,0 +1,240 @@
+﻿// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_EXT_FFT_INCLUDED_
+#define _NBL_EXT_FFT_INCLUDED_
+
+#include "nabla.h"
+#include "nbl/video/IGPUShader.h"
+#include "nbl/asset/ICPUShader.h"
+
+namespace nbl
+{
+namespace ext
+{
+namespace FFT
+{
+
+class FFT : public core::TotalInterface
+{
+	public:
+
```

Comment:
I'd probably add an option whether to use full `vec2` for storage of the intermediate data in the SSBO, or a packed half float (`pack2x16Half` and similar glsl builtins)